cmd/kube-controller-manager/app/core.go

```diff
@@ -244,6 +244,7 @@ func startVolumeExpandController(ctx ControllerContext) (http.Handler, bool, err
 			ctx.ClientBuilder.ClientOrDie("expand-controller"),
 			ctx.InformerFactory.Core().V1().PersistentVolumeClaims(),
 			ctx.InformerFactory.Core().V1().PersistentVolumes(),
+			ctx.InformerFactory.Storage().V1().StorageClasses(),
```

Comment:
Note for reviewers : ExpandController already had rbac permission to list/get storageclasses, so we don't need a authorization policy update.